README.md

```diff
@@ -1,6 +1,12 @@
 # Mozilla Rally study template.
-The purpose of this repository is to provide a starting point to build a Rally study.
-
+The purpose of this repository is to provide a starting point to build a Rally study. It contains:
+
+1. an example [`manifest.json'](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json) file.
+2. an example background and content script in `src`.
+3. an example options page in `public/index.html`. _note:_ this page is used in the provided integration test example.
```

Comment:
I thought of this a bit again: do we envision studies having option pages? Do we encourage them to? Do we want them to?

If not, then this page becomes one thing to chop off and a potential source of friction.

I'd recommend removing the option page depending on the discussion above. @hamilton ?